mls-rs/src/group/message_processor.rs

```diff
@@ -75,7 +75,7 @@ pub(crate) struct ProvisionalState {
     pub(crate) external_init_index: Option<LeafIndex>,
     pub(crate) indexes_of_added_kpkgs: Vec<LeafIndex>,
     #[cfg(all(feature = "state_update", feature = "by_ref_proposal"))]
-    pub(crate) rejected_proposals: Vec<ProposalInfo<Proposal>>,
+    pub(crate) rejected_proposals: Vec<crate::mls_rules::ProposalInfo<Proposal>>,
```

Comment:
Out of curiosity, why is the path fully qualified now?

Reply:
Because in a weird feature combination (the interop runner uses different ones than `cargo hack`) the import was not compiled. It was too difficult to figure out when to import :/

Reply:
Thank you for explaining.